app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
That's correct, I told @pattisdr to not use pod style for a project with a low number of components finding and deleting component files is not so much of a hassle and for me having several component.js file is more of a headache. But I wouldn't fight this if using pods is a better standard and beyond a matter of opinion.